distributed/scheduler.py

```diff
@@ -392,10 +404,19 @@ def start(self, port=8786, start_queues=True):
                     raise exc
 
         if self.status != 'running':
-            self.listen(port)
+            if isinstance(addr_or_port, int):
+                self.ip = get_ip()
+                # Listen on all interfaces.  `self.ip` is not suitable
+                # as its default value would prevent connecting via 127.0.0.1.
+                self.listen(('', addr_or_port))
+                self.start_services()
+            else:
+                self.listen(addr_or_port)
+                self.ip, _ = get_address_host_port(self.listen_address)
+                self.start_services(self.ip)
 
             self.status = 'running'
-            logger.info("  Scheduler at: %20s:%s", self.ip, self.port)
+            logger.info("  Scheduler at: %24s", self.address)
```

Comment:
This may not be aligned 

```
mrocklin@carbon:~$ dask-scheduler 
distributed.scheduler - INFO - -----------------------------------------------
distributed.scheduler - INFO -   Scheduler at:  tcp://10.22.112.88:8786
distributed.scheduler - INFO -        http at:              0.0.0.0:9786
distributed.scheduler - INFO -       bokeh at:              0.0.0.0:8788
distributed.bokeh.application - INFO - Web UI: http://127.0.0.1:8787/status/
distributed.scheduler - INFO - -----------------------------------------------
```